### PR TITLE
Update redirected URLs (part 44)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Extending #14055

## Summary

Updates redirected URLs.

### CODE REVIEW

File: files/en-us/web/guide/ajax/community/index.md [hunk 1]
```diff
@@ -14,5 +14,5 @@ If you know of useful mailing lists, newsgroups, forums, or other communities re
 - [telerik.com](https://www.telerik.com/forums/aspnet-ajax): An active community forum for Ajax
 - [community.tableau.com](https://community.tableau.com/search.jspa?q=ajax): Community support forum and courses available for Ajax
 - [codementor.io](https://www.codementor.io/community/search?q=ajax): Social platform with Ajax forums and tutorials
-- [lynda.com](https://www.lynda.com/search?q=ajax): Tutorials available for learning the fundamentals of Ajax
+- [lynda.com](https://www.linkedin.com/learning/search?trk=lynda_redirect_learning): Tutorials available for learning the fundamentals of Ajax
```
Comment (OnkarRuikar) [inline files/en-us/web/guide/ajax/community/index.md:17]:
```suggestion
- [LinkedIn Learning](https://www.linkedin.com/learning/search?keywords=ajax): Tutorials available for learning the fundamentals of Ajax
```

File: files/en-us/web/guide/audio_and_video_delivery/live_streaming_web_audio_and_video/index.md [hunk 2]
```diff
@@ -184,6 +184,6 @@ Although you can install software like GStreamer, SHOUTcast and Icecast you will
 - [GStreamer](https://en.wikipedia.org/wiki/GStreamer)
 - [Streaming GStreamer Pipelines Via HTTP](https://coaxion.net/blog/2013/10/streaming-gstreamer-pipelines-via-http/)
 - [GStreamer and Raspberry Pi](https://nginx-rtmp.blogspot.com/2013/07/gstreamer-and-raspberry-pi.html)
-- [Acceptance of Media Source Extensions as W3C Candidate Recommendation will accelerate adoption of dash.js](http://msopentech.com/blog/2014/01/09/acceptance-media-source-extensions-w3c-candidate-recommendation-will-accelerate-adoption-dash-js/)
+- [Acceptance of Media Source Extensions as W3C Candidate Recommendation will accelerate adoption of dash.js](https://opensource.microsoft.com/)
```
Comment (OnkarRuikar) [inline files/en-us/web/guide/audio_and_video_delivery/live_streaming_web_audio_and_video/index.md:187]:
```suggestion
```

File: files/en-us/web/guide/audio_and_video_delivery/setting_up_adaptive_streaming_media_sources/index.md [hunk 3]
```diff
@@ -110,7 +110,7 @@ it might be wise to provide a fallback for browsers that don't yet support MPEG-
 
 ### LIVE Profile
 
-A useful piece of software when dealing with MPEG-DASH is [Dash Encoder](https://github.com/slederer/DASHEncoder). This uses [MP4Box](https://github.com/gpac/gpac/wiki/MP4Box/dash/) to encode media into MPEG-DASH format.
+A useful piece of software when dealing with MPEG-DASH is [Dash Encoder](https://github.com/slederer/DASHEncoder). This uses [MP4Box](https://github.com/gpac/gpac/wiki/MP4Box) to encode media into MPEG-DASH format.
```
Comment (OnkarRuikar) [inline files/en-us/web/guide/audio_and_video_delivery/setting_up_adaptive_streaming_media_sources/index.md:113]:
```suggestion
A useful piece of software when dealing with MPEG-DASH is [Dash Encoder](https://github.com/slederer/DASHEncoder). This uses [MP4Box](https://github.com/gpac/gpac/wiki/mp4box-dash-opts) to encode media into MPEG-DASH format.
```

File: files/en-us/web/guide/introduction_to_web_development/index.md [hunk 4]
```diff
@@ -62,13 +62,13 @@ For another (overlapping) set of links to learning resources, see the [MDN Learn
   - : A reliable reference site for learning HTML, CSS and JavaScript which also mentions feature support across different browsers and known browser bugs.
 - [HTMLDog](https://htmldog.com/)
   - : An excellent and comprehensive reference site on HTML and CSS for beginners.
-- [Google's HTML, CSS, and Javascript from the Ground Up](https://code.google.com/edu/submissions/html-css-javascript/)
+- [Google's HTML, CSS, and Javascript from the Ground Up](https://developers.google.com/university)
```
Comment (OnkarRuikar) [inline files/en-us/web/guide/introduction_to_web_development/index.md:65]: > The University Consortium is no longer actively maintained. 
```suggestion
```

File: files/en-us/web/guide/introduction_to_web_development/index.md [hunk 5]
```diff
@@ -62,13 +62,13 @@ For another (overlapping) set of links to learning resources, see the [MDN Learn
...
   - : These easily digestible video tutorials from Google's expert web developers cover the basics of HTML, CSS and JavaScript.
```
Comment (OnkarRuikar) [inline files/en-us/web/guide/introduction_to_web_development/index.md:66]:
```suggestion
```

File: files/en-us/web/html/viewport_meta_tag/index.md [hunk 6]
```diff
@@ -79,7 +79,7 @@ Suppress the small zoom applied by many smartphones by setting the initial scale
 
 ## Common viewport sizes for mobile and tablet devices
 
-If you want to know what mobile and tablet devices have which viewport widths, there is a comprehensive list of [mobile and tablet viewport sizes here](https://docs.adobe.com/content/help/en/target/using/experiences/vec/mobile-viewports.html). This gives information such as viewport width on portrait and landscape orientation as well as physical screen size, operating system and the pixel density of the device.
+If you want to know what mobile and tablet devices have which viewport widths, there is a comprehensive list of [mobile and tablet viewport sizes here](https://experienceleague.adobe.com/docs/target/using/experiences/vec/mobile-viewports.html?lang=en). This gives information such as viewport width on portrait and landscape orientation as well as physical screen size, operating system and the pixel density of the device.
```
Comment (OnkarRuikar) [inline files/en-us/web/html/viewport_meta_tag/index.md:82]:
```suggestion
If you want to know what mobile and tablet devices have which viewport widths, there is a comprehensive list of [mobile and tablet viewport sizes here](https://experienceleague.adobe.com/docs/target/using/experiences/vec/mobile-viewports.html). This gives information such as viewport width on portrait and landscape orientation as well as physical screen size, operating system and the pixel density of the device.
```

File: files/en-us/web/http/basics_of_http/evolution_of_http/index.md [hunk 7]
```diff
@@ -181,7 +181,7 @@ The HTTP/2 protocol differs from HTTP/1.1 in a few ways:
 - It compresses headers. As these are often similar among a set of requests, this removes the duplication and overhead of data transmitted.
 - It allows a server to populate data in a client cache through a mechanism called the server push.
 
-Officially standardized in May 2015, HTTP/2 was incredibly successful. By July 2016, 8.7% of all websites used it (see [these stats](https://w3techs.com/technologies/details/ce-http2/all/all)), representing more than 68% of all requests (see [these statistics](https://www.keycdn.com/blog/http2-statistics)). High-traffic websites showed the most rapid adoption in an effort to save on data transfer overhead and subsequent budgets.
+Officially standardized in May 2015, HTTP/2 was incredibly successful. By July 2016, 8.7% of all websites used it (see [these stats](https://w3techs.com/technologies/details/ce-http2)), representing more than 68% of all requests (see [these statistics](https://www.keycdn.com/blog/http2-statistics)). High-traffic websites showed the most rapid adoption in an effort to save on data transfer overhead and subsequent budgets.
```
Comment (OnkarRuikar) [inline files/en-us/web/http/basics_of_http/evolution_of_http/index.md:184]:
```suggestion
Officially standardized in May 2015, HTTP/2 was incredibly successful. By May 2022, 46.4% of all websites used it (see [these stats](https://w3techs.com/technologies/details/ce-http2)). High-traffic websites showed the most rapid adoption in an effort to save on data transfer overhead and subsequent budgets.
```
Couldn't find source for latest request stats.

File: files/en-us/web/http/browser_detection_using_the_user_agent/index.md [hunk 8]
```diff
@@ -19,7 +19,7 @@ But browsers and standards are not perfect, and there are still some edge cases
...
+  - : Look, or ask, in specialized forums: you're unlikely to be the first to hit this problem. Also, experts, or people with another point of view, can give you ideas for working around the bug. If the problem seems uncommon, it's worth checking if this bug has been reported to the browser vendor via their bug tracking system ([Mozilla](https://bugzilla.mozilla.org); [WebKit](https://bugs.webkit.org); [Blink](https://www.chromium.org/issue-tracking/); [Opera](https://bugs.opera.com/login.jsp)). Browser makers do pay attention to bug reports, and the analysis may hint about other workarounds for the bug.
```
Comment (OnkarRuikar) [inline files/en-us/web/http/browser_detection_using_the_user_agent/index.md:22]:
```suggestion
  - : Look, or ask, in specialized forums: you're unlikely to be the first to hit this problem. Also, experts, or people with another point of view, can give you ideas for working around the bug. If the problem seems uncommon, it's worth checking if this bug has been reported to the browser vendor via their bug tracking system ([Mozilla](https://bugzilla.mozilla.org); [WebKit](https://bugs.webkit.org); [Blink](https://www.chromium.org/issue-tracking/); [Opera](https://bugs.opera.com/)). Browser makers do pay attention to bug reports, and the analysis may hint about other workarounds for the bug.
```

File: files/en-us/web/http/browser_detection_using_the_user_agent/index.md [hunk 9]
```diff
@@ -245,7 +245,7 @@ The following table summarizes the way common browser vendors indicate that thei
...
-| Presto-based (Opera 12-)                                          | `Opera Mobi/xyz` token [inside](https://my.opera.com/community/openweb/idopera/) the comment.                                                                                                                                                       | `Opera/9.80 (Android 2.3.3; Linux; Opera Mobi/ADR-1111101157; U; es-ES) Presto/2.9.201 Version/11.50`                                                            |
+| Presto-based (Opera 12-)                                          | `Opera Mobi/xyz` token [inside](https://blogs.opera.com/news/) the comment.                                                                                                                                                       | `Opera/9.80 (Android 2.3.3; Linux; Opera Mobi/ADR-1111101157; U; es-ES) Presto/2.9.201 Version/11.50`                                                            |
```
Comment (OnkarRuikar) [inline files/en-us/web/http/browser_detection_using_the_user_agent/index.md:248]:
```suggestion
| Presto-based (Opera 12-)                                          | `Opera Mobi/xyz` token [inside](https://developers.whatismybrowser.com/useragents/explore/layout_engine_name/presto/) the comment.                                                                                                                                                       | `Opera/9.80 (Android 2.3.3; Linux; Opera Mobi/ADR-1111101157; U; es-ES) Presto/2.9.201 Version/11.50`                                                            |
```

File: files/en-us/web/javascript/about_javascript/index.md [hunk 10]
```diff
@@ -33,7 +33,7 @@ Besides the above implementations, there are other popular JavaScript engines su
 
 - Google's [V8](https://v8.dev/), which is used in the Google Chrome browser and recent versions of Opera browser. This is also the engine used by [Node.js](https://nodejs.org).
 - The [JavaScriptCore](https://webkit.org/project/) (SquirrelFish/Nitro) used in some WebKit browsers such as Apple Safari.
-- [Carakan](https://my.opera.com/ODIN/blog/carakan-faq) in old versions of Opera.
+- [Carakan](https://dev.opera.com/blog/) in old versions of Opera.
```
Comment (OnkarRuikar) [inline files/en-us/web/javascript/about_javascript/index.md:36]:
```suggestion
- [Carakan](https://dev.opera.com/blog/carakan-faq/) in old versions of Opera.
```

File: files/en-us/web/javascript/reference/global_objects/intl/numberformat/numberformat/index.md [hunk 11]
```diff
@@ -237,14 +237,14 @@ new Intl.NumberFormat(locales, options)
       - : The minimum number of fraction digits to use. Possible values are from 0
         to 20; the default for plain number and percent formatting is 0; the
         default for currency formatting is the number of minor unit digits
-        provided by the [ISO 4217 currency code list](https://www.currency-iso.org/en/home/tables/table-a1.html)
+        provided by the [ISO 4217 currency code list](https://www.six-group.com/en/products-services/financial-information/data-standards.html)
```
Comment (OnkarRuikar) [inline files/en-us/web/javascript/reference/global_objects/intl/numberformat/numberformat/index.md:240]:
```suggestion
        provided by the [ISO 4217 currency code list](https://www.six-group.com/dam/download/financial-information/data-center/iso-currrency/lists/list_one.xml)
```

File: files/en-us/web/javascript/reference/global_objects/intl/numberformat/numberformat/index.md [hunk 12]
```diff
@@ -237,14 +237,14 @@ new Intl.NumberFormat(locales, options)
...
         (2 if the list doesn't provide that information).
     - `maximumFractionDigits`
       - : The maximum number of fraction digits to use. Possible values are from 0
         to 20; the default for plain number formatting is the larger of
         `minimumFractionDigits` and 3; the default for currency
         formatting is the larger of `minimumFractionDigits` and the
-        number of minor unit digits provided by the [ISO 4217 currency code list](https://www.currency-iso.org/en/home/tables/table-a1.html)
+        number of minor unit digits provided by the [ISO 4217 currency code list](https://www.six-group.com/en/products-services/financial-information/data-standards.html)
```
Comment (OnkarRuikar) [inline files/en-us/web/javascript/reference/global_objects/intl/numberformat/numberformat/index.md:247]:
```suggestion
        number of minor unit digits provided by the [ISO 4217 currency code list](https://www.six-group.com/dam/download/financial-information/data-center/iso-currrency/lists/list_one.xml)
```

File: files/en-us/web/javascript/reference/global_objects/intl/pluralrules/pluralrules/index.md [hunk 13]
```diff
@@ -63,14 +63,14 @@ new Intl.PluralRules(locales, options)
       - : The minimum number of fraction digits to use. Possible values are from 0
         to 20; the default for plain number and percent formatting is 0; the
         default for currency formatting is the number of minor unit digits
-        provided by the [ISO 4217 currency code list](https://www.currency-iso.org/en/home/tables/table-a1.html)
+        provided by the [ISO 4217 currency code list](https://www.six-group.com/en/products-services/financial-information/data-standards.html)
```
Comment (OnkarRuikar) [inline files/en-us/web/javascript/reference/global_objects/intl/pluralrules/pluralrules/index.md:66]:
```suggestion
        provided by the [ISO 4217 currency code list](https://www.six-group.com/dam/download/financial-information/data-center/iso-currrency/lists/list_one.xml)
```

File: files/en-us/web/javascript/reference/global_objects/intl/pluralrules/pluralrules/index.md [hunk 14]
```diff
@@ -63,14 +63,14 @@ new Intl.PluralRules(locales, options)
...
         (2 if the list doesn't provide that information).
     - `maximumFractionDigits`
       - : The maximum number of fraction digits to use. Possible values are from 0
         to 20; the default for plain number formatting is the larger of
         `minimumFractionDigits` and 3; the default for currency
         formatting is the larger of `minimumFractionDigits` and the
-        number of minor unit digits provided by the [ISO 4217 currency code list](https://www.currency-iso.org/en/home/tables/table-a1.html)
+        number of minor unit digits provided by the [ISO 4217 currency code list](https://www.six-group.com/en/products-services/financial-information/data-standards.html)
```
Comment (OnkarRuikar) [inline files/en-us/web/javascript/reference/global_objects/intl/pluralrules/pluralrules/index.md:73]:
```suggestion
        number of minor unit digits provided by the [ISO 4217 currency code list](https://www.six-group.com/dam/download/financial-information/data-center/iso-currrency/lists/list_one.xml)
```

File: files/en-us/web/javascript/reference/operators/delete/index.md [hunk 15]
```diff
@@ -54,7 +54,7 @@ Throws {{jsxref("TypeError")}} in [strict mode](/en-US/docs/Web/JavaScript/Refer
 ## Description
 
 Unlike what common belief suggests (perhaps due to other programming languages like
-[delete in C++](https://docs.microsoft.com/cpp/cpp/delete-operator-cpp?view=vs-2019)),
+[delete in C++](https://docs.microsoft.com/en-us/cpp/cpp/delete-operator-cpp?view=msvc-170&viewFallbackFrom=vs-2019)),
```
Comment (OnkarRuikar) [inline files/en-us/web/javascript/reference/operators/delete/index.md:57]:
```suggestion
[delete in C++](https://docs.microsoft.com/en-us/cpp/cpp/delete-operator-cpp?view=msvc-170)),
```

**Review by teoli2003 — APPROVED** [review]
LGTM 👍

**Pull request merged**